tests/test_schema.py

```diff
@@ -653,9 +653,6 @@ def test_only_in_init(SchemaClass, user):
     assert 'name' in s
     assert 'age' in s
 
-def test_invalid_only_param(user):
```

Comment:
Why was this test removed? Does this change introduce backwards-incompatible behavior?

Reply:
Yes there is a minor change. The current behaviour of only is not consistent. To make it consistent I ignored all fields passed in with only which are not present on the schema. This affected two tests.

1. test_only_with_invalid_attribute
2. test_invalid_only_param

Both pass in a value for `only` which is not present in the schema and then expect AttributeError to be raised because those fieds are not present in the model being dumped. With these changes, those fields which are not present on the schema will be ignored, therefore no erros will be raise and the tests will fail.

For fields present on the Schema but not in the model the behavior remains the same.